tests/ammo_set_test.cpp

```diff
@@ -74,14 +74,20 @@ TEST_CASE( "ammo_set", "[ammo_set][magazine][ammo]" )
                     CHECK( cz75mag_20rd.ammo_current().str() == ammo9mmfmj_id.str() );
                 }
             }
-            // WHEN( "set 308 ammo in the 9mm magazine" ) {
-            //     itype_id ammo308_id( "308" );
-            //     cz75mag_20rd.ammo_set( ammo308_id, 15 );
-            //     THEN( "magazine has 0 round of null" ) {
-            //         CHECK( cz75mag_20rd.ammo_remaining() == 0 );
-            //         CHECK( cz75mag_20rd.ammo_current().is_null() );
-            //     }
-            // }
+            WHEN( "set 308 ammo in the 9mm magazine" ) {
+                itype_id ammo308_id( "308" );
+                std::string dmsg = captureDebugmsgLambda( [&cz75mag_20rd, &ammo308_id]() {
+                    cz75mag_20rd.ammo_set( ammo308_id, 15 );
+                } );
+                THEN( "get debugmsg with \"Tried to set invalid ammo of 308 for cz75mag_20rd\"" ) {
+                    REQUIRE( !dmsg.empty() );
```

Comment:
```suggestion
```
I think this is redundant.

Reply:
(and similarly for the other test cases)

Reply:
done